ISISScientificComputing/autoreduce-frontend
# 1516 next previous runs ar 1516 ar 1517

### Summary of work
Uses django-next-prev to obtain next and previous objects from a querset for run navigation buttons, instead of just +1/-1 on the run number.

Checks "sort" query in URL to sort queryset based on run number or date. Fetches next/prev items based on this.

### How to test your work
1. Navigate to http://127.0.0.1:8000/runs/LARMOR/
2. Change the “Sort by” select box to “Date” and then apply the filter.
3. Click on run number 60197
4. Adjacent run buttons should be for runs 60199 and 60196

### Additional comments
Makes use of django-next-prev so might require `pip install django-next-prev` if not fresh setup
